azure-devops/projects/selfcare-projects/project.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  azure_devops_org = "pagopaspa"
+}
+
+resource "azuredevops_project" "project" {
+  name               = "pagoPA-projects"
+  description        = "This is the DevOps project for pagoPA service projects"
+  visibility         = "public"
+  version_control    = "Git"
+  work_item_template = "Basic"
+}
+
+resource "azuredevops_project_features" "project-features" {
+  project_id = azuredevops_project.project.id
+  features = {
+    "boards"       = "disabled"
+    "repositories" = "disabled"
+    "pipelines"    = "enabled"
+    "testplans"    = "disabled"
+    "artifacts"    = "enabled"
```

Comment:
can we not use artifacts inside devops?

Reply:
I think that will be used common libraries that need to be deployed and re-used in other devops builds of the current project